vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/dialog/tests/DialogTestPageIT.java

```diff
@@ -63,9 +63,8 @@ public void dialogWithOpenedChangeListener() {
 
         Assert.assertEquals("The open state of the dialog is true",
                 message.getText());
-        Assert.assertEquals(
-                "There should not be initial events before opening the dialog",
-                "Number of event is 0", eventCounterMessage.getText());
+        Assert.assertEquals("There should one event from opening the dialog",
+                "Number of event is 1", eventCounterMessage.getText());
```

Comment:
Not sure if I'm missing something, but opening the dialog should fire an opened change event?

Reply:
Yeah OK, there actually was a bug in that just adding a closed dialog to the DOM causes an opened change event. Added a fix here: https://github.com/vaadin/flow-components/pull/5258/commits/538ba4b3e37c0d49f5f6f978e45d102056e0d0cf

Still kept the updated assertions, so that the first actual change event changes the counter to `1`, rather then kepp it at `0`.